pandas/io/stata.py

```diff
@@ -1897,7 +1925,8 @@ def _convert_datetime_to_stata_type(fmt):
         raise NotImplementedError(f"Format {fmt} not implemented")
 
 
-def _maybe_convert_to_int_keys(convert_dates, varlist):
+# TODO
+def _maybe_convert_to_int_keys(convert_dates: Dict, varlist) -> Dict:
```

Comment:
is the todo to add type parameters to Dict?

Reply:
Yes, needed that one more.